packages/netlify-cms-lib-util/src/APIUtils.ts

```diff
@@ -2,10 +2,15 @@ export const CMS_BRANCH_PREFIX = 'cms';
 export const DEFAULT_PR_BODY = 'Automatically generated by Netlify CMS';
 export const MERGE_COMMIT_MESSAGE = 'Automatically generated. Merged on Netlify CMS.';
 
-const NETLIFY_CMS_LABEL_PREFIX = 'netlify-cms/';
-export const isCMSLabel = (label: string) => label.startsWith(NETLIFY_CMS_LABEL_PREFIX);
-export const labelToStatus = (label: string) => label.substr(NETLIFY_CMS_LABEL_PREFIX.length);
-export const statusToLabel = (status: string) => `${NETLIFY_CMS_LABEL_PREFIX}${status}`;
+const DEFAULT_NETLIFY_CMS_LABEL_PREFIX = 'netlify-cms/';
+const getLabelPrefix = (labelPrefix: string) => labelPrefix || DEFAULT_NETLIFY_CMS_LABEL_PREFIX;
+
+export const isCMSLabel = (label: string, labelPrefix = '') =>
```

Comment:
I recommend removing the default value for `labelPrefix`, then running `yarn test` to see all the places you should be passing the prefix